styles/newsletter.scss

```diff
@@ -12,92 +10,98 @@ $o-grid-gutter-small: oGridGutter(S);
 	flex-direction: column;
 	position: relative;
 	width: 100%;
+	background-color: oColorsByName("white-80");
 	border-bottom: 1px solid oColorsByName('black-20');
-}
 
-.n-newsletter-signup__header {
-	display: flex;
-	position: relative;
-	box-sizing: border-box;
-	min-height: 100px;
-	padding: 5px 12px;
-	background-color: oColorsByName('claret');
-	color: white;
-}
-
-.n-newsletter-signup__heading {
-	@include oTypographyDisplay($scale: 2, $weight: 'bold');
-
-	display: inline-block;
-	align-self: flex-end;
-	margin: 0;
-
-	// for IE<11
-	max-width: 100%;
-	flex-shrink: 1;
-	padding-top: 20px;
-	// 😢
-	font-size: 30px;
-	line-height: 30px;
-
-	@include oGridRespondTo($from: M, $until: XL) {
-		font-size: 24px;
-		line-height: 28px;
+
+	.n-newsletter-signup__header {
+		display: flex;
+		position: relative;
+		box-sizing: border-box;
+		min-height: 100px;
+		padding: 5px 12px;
+		background-color: oColorsByName('claret');
+		color: white;
```

Comment:
**Change request:** Here can we use `oColorsByName` or maybe a hex code `#ffffff` instead of `white`

Reply:
@Nazehs This is still waiting to be actioned